src/components/dragons.js

```diff
@@ -9,7 +9,19 @@ const Dragons = () => {
 
   useEffect(() => {
     dispatch(fetchDragonsAsync());
-  });
+  }, [dispatch]);
```

Comment:
Please, move this section to a different file, so the API does not get called again every time it renders this file. I'd suggest moving it to the app.js file.